lampod-cli/src/args.rs

```diff
@@ -65,11 +65,6 @@ impl TryInto<LampoConf> for LampoCliArgs {
         };
 
         let path = self.data_dir.unwrap_or(conf.root_path);
-        // if the path doesn't exist, return an error
-        if !std::path::Path::new(&path).exists() {
-            error::bail!("The path {} doesn't exist", path);
-        }
```

Comment:
Mh I am wondering if we should check if the directory exist, and if not we will create it?

I am sure that for the network directory inside the `.lampo` we do the same

Reply:
Have you tried running this branch? That's what it's doing :)

Reply:
Here's the line where it happens https://github.com/vincenzopalazzo/lampo.rs/blob/main/lampo-common/src/conf.rs#L70